spk/radarr/src/wizard/upgrade_uifile_fre

```diff
@@ -1,7 +1,9 @@
 [{
     "step_title": "Mettre à jour Radarr",
     "items": [{
-        "desc": "Garder Radarr à jour en utilisant System>Updates dans l'interface Radarr."
+        "desc": "This update is for additional libraries and DSM integration. It does not modify your current Radarr version."
```

Comment:
@th0ma7 @ymartin59 kindly requesting a french translation

Reply:
IMO the french translation by google translator does not look bad...

Reply:
@hgy59 sorry for not answering earlier, was afk.  Indeed, translation looks ok to me as well.

Reply:
> @hgy59 sorry for not answering earlier, was afk. Indeed, translation looks ok to me as well.

No problem, 
_it was a little odyssey last weekend to complete the rework of radarr package...._